dockerfiles/diy/build_docker_image.sh

```diff
@@ -131,9 +131,8 @@ if [ "${BUILD_IMAGE}" = true ]; then
 else
     # Let the user know what we are going to do
     echo "Building image: ${ROUTER_VERSION}" from released tarballs""
-    ROUTER_RELEASE="$(echo "${ROUTER_VERSION}" | cut -c2-)"
```

Comment:
In the same spirit as the change in the `.circleci/config.yml` above, this transformation of the `ROUTER_VERSION` to remove the `v` character is no longer necessary and we can treat this like an opaque value.